src/representative.jl

```diff
@@ -9,17 +18,106 @@ used for each representative period. In addition, each representative period
 has an associated share that specifies how much of the total duration that
 is attributed to it.
 
+!!! note
+    - All representative periods must use the same type for the `TimeStructure`.
+    - If the field `period_share` is not specified, it assigns the same probability to each
+      representative period.
+    - It is possible that `sum(period_share)` is larger or smaller than 1. This can lead to
+      problems in your application. Hence, it is advised to scale it. We provide currently a
+      warning as it would correspond to a breaking change.
```

Comment:
What would be a breaking change? Automatic scaling? I suggest rephrasing to make it clear.

Reply:
It could lead to a situation in which the operational vs strategic impacts in a model, _e.g._, CAPEX _vs._ variable OPEX is significantly changed in the model results. Although I do not see a reason for an application that does not sum up to 1, there may be one.

It would be breaking in the sense of altering results, not in the sense of altering the API.

Reply:
I also think the sentence could be rephrased slightly. E.g. "Currently, a warning will be given if the period shares do not sum to one as an automatic scaling will correspond to a breaking change."